docs/tutorials/pre_executed/timeseries.ipynb

```diff
@@ -0,0 +1,1471 @@
+{
```

Comment:
<p><code>the probability of a false alarm</code> -&gt; <code>the false-alarm probability that the periodogram peak arose from noise rather than a true signal</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/astronomy-commons/lsdb/pull/834/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='24'/>